dist/scripts/api-console.js

```diff
@@ -1725,7 +1732,7 @@
 
   angular.module('raml', [])
     .factory('ramlParser', function () {
-      return RAML.Parser;
+      return RAML1Parser;
```

Comment:
The creation of the global RAML was done in the old parser, I need to do the same when packaging the new parser using webpack. I need to investigate about webpack a bit more to understand how to do this.